### PR TITLE
Adds tech shells to the outpost market, recipe rebalance and buffs

### PR DESCRIPTION
**Pull request by Gristlebee**
## About The Pull Request

Adds unloaded techshells to the outpost market, 7 shells for 175 credits.

### Recipe changes

- Scatter Laser: 1 Adv Capacitor, 1 Adv Microlaser -> 1 Capacitor, 3 Microlaser, 1 power cell, 5u lithium
- Pulse Slug: 2 Adv Capacitor, 1 Ultra Microlaser -> 3 Capacitor, 1 Microlaser, 1 power cell, 5u lithium
- Scatter Ion: 1 Ansible Crystal, 1 Ultra Microlaser -> 2 Capacitor, 2 Microlaser, 1 scanning module, 5u iron, 5u uranium
- Dragonsbreath : 5u phosphorous -> 10u phosphorous

Changes the scatter ion desc to reflect the new recipe.

### Ammo changes

Scatter laser and ion now fires 8 pellets, instead of 4, variance is 25 from 35
Dragonsbreath now fires 8 pellets instead of 4, deals 8 damage up from 5, variance is 45 from 35.

## Why It's Good For The Game

These techshells were pretty underwhelming for what they were, and pretty heavily gated behind RND. For all the effort of researching and printing techshells with scatter lasers for example, you'd get a 60 damage shell on par with an improvised buckshot you made with scrap.

These changes should make them more comparable with standard ammunition and not so reliant on needing RND. I think techshells are pretty interesting and adds a versatile combat option.

## Changelog

:cl:
add: Techshell box
add: Techshells to the outpost market for 175 credits
tweak: Scatter ion description
balance: Techshell recipes no longer require higher tier stock parts, adjusts scatter laser, scatter ion, dragonsbreath and pulse slug recipes
balance: Buffs scatter lasers, scatter ion and dragonsbreath
/:cl:

### CODE REVIEW

**Comment by thgvr** [comment]
personally i think tech shells are lame and would rather see them removed

**Comment by Imaginos16** [comment]
Personally this is a goated as hell PR and it gives tech shells a proper niche now instead of it being gatekept by RND shenanigans. If only the other RND-gated Roumain guns could follow suit.

**Comment by Imaginos16** [comment]
>If only the other RND-gated Roumain guns could follow suit.

Actually this is quite the idea for a PR now that I think about it, time to cook.

**Comment by thgvr** [comment]
I also don't really want to see pulse slugs taking down walls as well as the super turf fires

**Comment by Gristlebee** [comment]
> I also don't really want to see pulse slugs taking down walls as well as the super turf fires

Pulse shots dont break walls anymore after #2315. The turf fires from the pulse slug is also handled by a proc, so I can tone down it's strength if that's an issue.

File: code/modules/projectiles/ammunition/ballistic/shotgun.dm [hunk 1]
```diff
@@ -127,20 +127,20 @@
...
-	pellets = 4
-	variance = 35
+	pellets = 8
+	variance = 25
 
 /obj/item/ammo_casing/shotgun/laserscatter
 	name = "scatter laser shell"
 	desc = "An advanced shotgun shell that uses a micro laser to replicate the effects of a scatter laser weapon in a ballistic package."
 	icon_state = "laser"
 	projectile_type = /obj/projectile/beam/weak
```
Comment (SomeguyManperson) [inline code/modules/projectiles/ammunition/ballistic/shotgun.dm:141]: 15 damage -5 AP compared to buckshot 13 damage -35 AP

Comment (Gristlebee) [reply code/modules/projectiles/ammunition/ballistic/shotgun.dm:141]: Did some testing, and against a standard armor vest, an individual weak beam deals about 10.3 damage vs a buckshot pellet dealing 7. So about 83 damage laser scatter  vs 56 damage from buckshot assuming every shot hits. It's a decent chunk of extra damage, but not a one hit crit.

Comment (Apogee-dev) [reply code/modules/projectiles/ammunition/ballistic/shotgun.dm:141]: I wouldnt compare laser shells to buckshot personally. In terms of niche theyre closer to slugs as an anti-armor option. Which they should definitely outclass at point blank range but I’d like to see how they stack up at different ranges.

File: code/modules/cargo/packs/ammo.dm [hunk 2]
```diff
@@ -70,6 +70,12 @@
 	cost = 500
 	contains = list(/obj/item/ammo_box/a12g/slug)
 
+/datum/supply_pack/ammo/techshells
+	name = "Unloaded Shotgun Technological Shells Crate"
+	desc = "Contains a box of 7 versatile tech shells, capable of producing a variety of deadly effects for any situation. Some assembly required."
+	cost = 175
```
Comment (FalloutFalcon) [inline code/modules/cargo/packs/ammo.dm]: 20 per buckshot or slug
25 per tech shell..
I would prefer if they cost more, the crafting ingredients aren't enough to affect the balance

**Review by FalloutFalcon — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs/ammo.dm (hunk 2 above)
Comment (Gristlebee) [reply code/modules/cargo/packs/ammo.dm]: upped it by another 5, it's now 30 per tech shell.

**Review by FalloutFalcon — APPROVED** [review]
Ill need thgvrs approval on this one since they had thoughts about tech shells but i support it

**Comment by Imaginos16** [comment]
Thgvr said they didn't particularly care so I'll get this merged. Yippie!

**Pull request merged**